src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/documentmapper/PartADocumentMapper.kt

```diff
@@ -94,8 +95,7 @@ class PartADocumentMapper : RecommendationDataToDocumentMapper() {
       mappa = recommendation.personOnProbation?.mappa,
       lastRecordedAddress = lastRecordedAddress,
```

Comment:
What's the intention here?
There are several lines in this block of code that are doing similar (e.g. 96, 129, 130).

Reply:
Changed my mind.

Reply:
Each line sets a different field.
96 sets the lastRecordedAddress
126 sets the counterSignAcoNAme
130 sets the countersignSpoTime

The purpose of the PartADocumentMapper is to take data from the Recommendation journey (living in the status and rec doc tables). And create a DocumentData object which is used to populate the fields in the word doc.